src/main/java/com/ifactory/press/db/solr/HitCount.java

```diff
@@ -61,7 +61,7 @@ public ValueSource parse(FunctionQParser fp) throws SyntaxError {
                 https://issues.apache.org/jira/browse/LUCENE-6425
              */
             IndexReader emptyReader = new MultiReader();
-            new IndexSearcher(emptyReader).createNormalizedWeight(q, true).extractTerms(terms);
+            //new IndexSearcher(emptyReader).createNormalizedWeight(q, true).extractTerms(terms);
         } catch (UnsupportedOperationException e) {
```

Comment:
this may be needed? and could be why test fails?

Reply:
Yes this changed to createWeight and it has to take a booster